docker/README.md

```diff
@@ -0,0 +1,12 @@
+Open project path in Terminal and run below commands; Tested with MacPro M1.
+
+- NAME = Docker image name
+- TAG = Docker image tag, used package.json > version
+```
+export NAME=yavuzslmtpl/console-remote-server
+export TAG=2.1.17
+# For local
+docker build -f docker/Dockerfile -t ${NAME}:${TAG} --no-cache .
+# For push
+docker buildx build -f docker/Dockerfile -t ${NAME}:${TAG},${NAME}:latest --platform=linux/amd64,linux/arm64 --push .
```

Comment:
I would include command for running the container locally to confirm that this whole setup works as expected... I ran everything as is and i'm not able to reach the server locally